scripts/GenSongsList.py

```diff
@@ -93,30 +78,14 @@ def findMatchingBasename(files, basename):
 #       else:
 #         htmlOutput.write(f"  <a href=\"{str(i)}\">{ext(i)}</a>\n")
 #
-        if ext(i).lower() == ".pdf":
-          htmlOutput.write(f"  <a href=\"{str(i)}\">{ext(i)}</a>\n")
-        elif ext(i).lower() == ".chopro":
-          htmlOutput.write(f"  <a href=\"{str(i)}\" download>{ext(i)}</a>\n")
-        elif ext(i).lower() == ".cho":
+        if ext(i) == ".url":
+          with open(i, "r") as urlFile:
+            address = urlFile.readline()
+          htmlOutput.write(f"<a href=\"{address}\">video</a>\n")
```

Comment:
Here's the magic for using the contents of a ".url" file to create a "video" link. The .url file could point anywhere, not necessarily to a video. We should probably discuss whether the word "video" should be something more generic like "external link".